src/assets/sass/_header.scss

```diff
@@ -0,0 +1,109 @@
+@media only screen and (min-width: 769px){
+    .header-container {
+        display: block;
+    }
+}
+.header-container .header-main {
+    color: #0b0c0c;
+    padding: 0.5rem 0px;
+    border-bottom: 6px solid #008938;
+    background: #FFF !important;
+}
+
+.header-div-container {
+    width: 100%;
+}
+
+.header-container .header-main .site-width-container {
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+}
+
+@media only screen and (min-width: 1020px) {
+    .header-div-container .site-width-container {
+        width: 100%;
```

Comment:
```suggestion
    As we haven't mentioned any width for its parent, by default this will take up the full width. So we can remove this. However, having this will not cause any harm.
```

Reply:
As per our discussion, there is no harm in having this. We can ignore this.

Reply:
we must keep it as we are using box model. the width covers upto the border area.